bangazonapi/views/profile.py

```diff
@@ -185,7 +185,6 @@ def cart(self, request):
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={
                                                 'request': request}).data
-                cart["order"]["line_items"] = line_items.data
```

Comment:
Nailed it! This line of code is all that needed to be removed in order to resolve issue ticket #24